docs/Admin.md

```diff
@@ -524,7 +523,7 @@ function AsyncResources() {
     }, []);
 
     return (
-        <AdminUI ready={Loading}>
+        <AdminUI ready={() => <span />}>
```

Comment:
I don't get why you change it by a `span`, It seems to work with the `Loading` component